guides/common/assembly_configuring-satellite-custom-server-certificate.adoc

```diff
@@ -4,16 +4,31 @@ ifdef::context[:parent-context: {context}]
 = Configuring {ProjectServer} with a Custom SSL Certificate
 
 By default, {ProjectName} uses a self-signed SSL certificate to enable encrypted communications between {ProjectServer}, external {SmartProxyServers}, and all hosts.
-If you cannot use a {Project} self-signed certificate, you can configure {ProjectServer} to use an SSL certificate signed by an external Certificate Authority.
+If you cannot use a {Project} self-signed certificate, you can configure {ProjectServer} to use an SSL certificate signed by an external certificate authority (CA).
+
+When you configure {ProjectName} with custom SSL certificates, you must fulfill the following requirements:
+
+* You must use the privacy-enhanced mail (PEM) encoding for the SSL certificates.
+* You must not use the same SSL certificate for both {ProjectServer} and {SmartProxyServer}.
+* The same CA must sign certificates for {ProjectServer} and {SmartProxyServer}.
+* An SSL certificate must not also be a CA certificate.
+* An SSL certificate must include a subject alt name (SAN) entry that matches the common name (CN).
+* An SSL certificate must be allowed for Key Encipherment using a Key Usage extension.
+* An SSL certificate must not have a shortname as the CN.
+* You must not set a passphrase for the private key.
 
 To configure your {ProjectServer} with a custom certificate, complete the following procedures:
 
+. xref:creating-a-custom-ssl-certificate_{project-context}[]
 . xref:Deploying_a_Custom_SSL_Certificate_to_Server_{project-context}[]
 . xref:deploying-a-custom-ssl-certificate-to-hosts_{project-context}[]
 . If you have external {SmartProxyServers} registered to {ProjectServer}, you must configure them with custom SSL certificates.
-The same Certificate Authority must sign certificates for {ProjectServer} and {SmartProxyServer}.
+The same certificate authority (CA) must sign certificates for {ProjectServer} and {SmartProxyServer}.
```

Comment:
```suggestion
The same CA must sign certificates for {ProjectServer} and {SmartProxyServer}.
```
Although I'm not sure why we're repeating this sentence when we already have it in the above requirements.

Reply:
Omitted :+1: